### PR TITLE
fix(chat): add messages for external files sharing errors

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- add error message for publish action
- add error message for share action

Issues:

- Issue #1984 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2020.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2020.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/store/share/share.epics.ts [hunk 1]
```diff
@@ -252,8 +252,13 @@ const sharePromptFolderEpic: AppEpic = (action$) =>
 const shareFailEpic: AppEpic = (action$) =>
   action$.pipe(
     filter(ShareActions.shareFail.match),
-    map(() => {
-      return UIActions.showErrorToast(translate(errorsMessages.shareFailed));
+    map(({ payload }) => {
+      const msg =
+        payload?.toLowerCase()?.trim() === 'bad request'
```
Comment (IlyaBondar) [inline apps/chat/src/store/share/share.epics.ts]: it's not good that we doesn't have specific error for sharing of public attachment. Only common `bad request`. Need to connect to Core-team to fix it

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2020.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**